requirements-dev.txt

```diff
@@ -0,0 +1,10 @@
+humanfriendly==10.0
+opencv_python>=4.8.0.74,<=4.9.0.80
+Pillow==10.3.0
+pillow_avif_plugin>=1.3.1,<=1.4.3
+PySide6>=6.5.1.1,<=6.6.3.1
+PySide6_Addons>=6.5.1.1,<=6.6.3.1
+PySide6_Essentials>=6.5.1.1,<=6.6.3.1
+typing_extensions>=3.10.0.0,<=4.11.0
+ujson>=5.8.0,<=5.9.0
```

Comment:
```suggestion
```

There was probably some misunderstanding, but these dependencies should remain only in `requirements.txt`, otherwise these files will get out of sync sooner or later and it will cause problems.

If you really want to have everything installed via a single dependency file, put here `-r requirements.txt` isntead. Otherwise I think both these files should be independent from each other. Ie. `requirements.txt` for installing production dependencies, `requirements-dev.txt` for developers' dependencies.